.chloggen/mackjmr-deprecate-usePreviewRules.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component (e.g. pkg/quantile)
+component: pkg/otlp/attributes
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Deprecate and replace functions which have usePreviewRules parameter.
```

Comment:
I think it would be helpful to include some context here on why we're making this change (i.e. preparing to graduate the `exporter.datadog.hostname.preview` feature gate to stable)

Reply:
Added the context 🙂